cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/distributed/DistributedSparkProgramRunner.java

```diff
@@ -174,7 +174,7 @@ protected void setupLaunchConfig(ProgramLaunchConfig launchConfig, Program progr
 
     launchConfig
       .addExtraEnv(extraEnv)
-      .addExtraDependencies(SparkProgramRuntimeProvider.class)
+      .addExtraDependencies(SparkProgramRuntimeProvider.class, SparkTwillRunnable.class)
```

Comment:
Is this change needed? If it is, I'm a little confused how the existing code path works without SparkTwillRunnable.class as a dependency.